torch/jit/__init__.py

```diff
@@ -1741,6 +1741,10 @@ def _unwrap_optional(x):
     assert x is not None, "Unwrapping null optional"
     return x
 
+# primitive that returns True when in compilation and False otherwise
+def _is_scripting():
```

Comment:
If you intend to use this in library code like torchvision, it should be considered part of our public interface (and thus not prefixed by `_`).